config/application.rb

```diff
@@ -1,5 +1,12 @@
 require_relative 'boot'
-require 'rails/all'
+
+require 'active_record/railtie'
```

Comment:
Was getting errors related to active storage not being configured when it was required by `rails/all`. We don't use it (or some of the other things like action cable), so I've only included the ones we use: https://guides.rubyonrails.org/initialization.html#railties-lib-rails-all-rb

Reply:
Big fan of explicit requires like this!